klaytnetl/mappers/receipt_mapper.py

```diff
@@ -144,3 +152,13 @@ def receipt_to_dict(self, receipt: KlaytnRawReceipt) -> dict:
             "max_priority_fee_per_gas": receipt.max_priority_fee_per_gas,
             "max_fee_per_gas": receipt.max_fee_per_gas,
```

Comment:
Don't we need to add `block_timestamp` here?

Reply:
we only add when enrich is enabled.